Support building Zipkin on ARM64

Hi Team,

I am interested in adding ARM64 support to Zipkin. 

There is an ‘elasticsearch’ binary for x86_64, getting downloaded in the RUN clause of the Dockerfile in zipkin/docker/storage/elasticsearch7/Dockerfile. The binary is available for aarch64 as well. I tried replacing the binary with aarch64, but build fails for zipkin-cassandra:2.21.5 docker image.

Please find my ARM64 job here : https://travis-ci.com/github/odidev/zipkin-1/jobs/368528378

Here is the failure logs : 
~~~
2020-08-05 06:53:00:414 [main] WARN zipkin2.storage.cassandra.v1.CassandraStorageExtension - Couldn't connect to docker image openzipkin/zipkin-cassandra:2.21.5: Assumption failed: All host(s) tried for query failed (tried: /127.0.0.1:9042 (com.datastax.driver.core.exceptions.TransportException: [/127.0.0.1:9042] Cannot connect))
~~~
I have checked in the docker hub <https://hub.docker.com/r/openzipkin/zipkin-cassandra/tags> that zipkin-cassandra is only available for LINUX/AMD64. 
I am suspecting that the further issues I am getting in the ARM64 build is due to zipkin-cassandra docker image unavailable for ARM64.
~~~
[ERROR]  HttpCallTest.wrongScheme:224
 
Expecting throwable message:
 <"not an SSL/TLS record: 485454502f312e31203430302042616420526571756573740d0a636f6e74656e742d747970653a20746578742f706c61696e3b20636861727365743d7574662d380d0a636f6e74656e742d6c656e6774683a2033310d0a7365727665723a2041726d657269612f302e39392e380d0a646174653a205468752c203330204a756c20323032302031353a35323a333720474d540d0a0d0a3430302042616420526571756573740a4465636f646572206661696c757265">
to contain:
 
 <"OPENSSL_internal">
 
but did not.
~~~
It would be really helpful if zipkin-cassandra docker image becomes available for LINUX/ARM64 platform as well. 
Please share your opinion on this.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,6 @@
 # Run `travis lint` when changing this file to avoid breaking the build.
 
 # See https://docs.travis-ci.com/user/reference/overview/#for-a-particular-travisyml-configuration
-arch: amd64 # Not arm64 for multi-platform Docker builds (qemu is amd64->arm64, not vice versa)
 os: linux   # required for arch different than amd64
 dist: focal # newest available distribution
 
@@ -23,6 +22,9 @@ services:
 
 # Re-use JDK 11 from focal release instead of delaying build with installation
 before_install:
+  - if [ "${TRAVIS_CPU_ARCH}" == "arm64" ]; then
+     sudo apt-get install -y maven;
+    fi
   - |
     # Cache as help:evaluate is not quick
     export POM_VERSION=$(./mvnw help:evaluate -N -Dexpression=project.version -q -DforceStdout)
@@ -72,9 +74,19 @@ install: ./build-bin/go_offline
 jobs:
   include:
     - name: default
+      arch: amd64 # Not arm64 for multi-platform Docker builds (qemu is amd64->arm64, not vice versa)
       script: ./travis/publish.sh
       if: 'NOT tag =~ /^docker-/'
+    - name: default
+      arch: arm64
+      script:
+        - |
+          # verify runs both tests and integration tests (Docker tests included)
+          # -DskipActuator ensures no tests rely on the actuator library
+          ./mvnw verify -nsu -DskipActuator
+      if: 'NOT tag =~ /^docker-/'
     - name: Re-push all Docker images
+      arch: amd64 # Not arm64 for multi-platform Docker builds (qemu is amd64->arm64, not vice versa)
       script:
         - |
           docker/bin/push_all_images $(echo "${TRAVIS_TAG}" | sed 's/^docker-//') &&

